src/box/tuple.h

```diff
@@ -78,13 +78,14 @@ tuple_arena_destroy(struct slab_arena *arena);
 /**
  * Creates a new format for standalone tuples.
  * Tuples created with the new format are allocated from the runtime arena.
- * In contrast to the preallocated tuple_format_runtime, which has no field
- * names, the new format uses the provided field name dictionary.
+ * In contrast to the preallocated tuple_format_runtime, which has no
+ * information about fields, the new format uses the provided field definitions.
  *
  * On success, returns the new format. On error, returns NULL and sets diag.
  */
 struct tuple_format *
-runtime_tuple_format_new(struct tuple_dictionary *dict);
+runtime_tuple_format_new(const struct field_def *fields, uint32_t field_count,
+			 bool is_external);
```

Comment:
What does `is_external` mean in terms of tuple formats?

Reply:
That they come from IPROTO.

Reply:
If it is possible to describe in the flag name what exactly makes a tuple format created with this flag different, I think, it would simplify understanding. Say, `names_only` feels much more obvious.

No insisting from my side, just attempt to share my feelings.

It is tightly connected with [this discussion](https://github.com/tarantool/tarantool/pull/8549#discussion_r1200841253) regarding a name of the function that creates such external formats from Lua.